### ISSUE TITLE
Conjure subprojects don't mark src/generated/java as a source root in intellij

### ISSUE THREAD

**Issue by tjwilson90** [palantir/gradle-conjure#259]
On 4.13.1, I get iml files with
```
<content url="file://$MODULE_DIR$/">
      <sourceFolder url="file://$MODULE_DIR$/generated_src" isTestSource="false" generated="true"/>
      <sourceFolder url="file://$MODULE_DIR$/src/generated/java" isTestSource="false" generated="true"/>
      <sourceFolder url="file://$MODULE_DIR$/generated_testSrc" isTestSource="true" generated="true"/>
      <excludeFolder url="file://$MODULE_DIR$/.gradle"/>
      <excludeFolder url="file://$MODULE_DIR$/build"/>
      <sourceFolder url="file://$MODULE_DIR$/generated_src" isTestSource="false" generated="true"/>
      <sourceFolder url="file://$MODULE_DIR$/generated_testSrc" isTestSource="true" generated="true"/>
</content>
```
On 4.13.3 (the latest release) I get
```
<content url="file://$MODULE_DIR$/">
      <excludeFolder url="file://$MODULE_DIR$/.gradle"/>
      <excludeFolder url="file://$MODULE_DIR$/build"/>
      <sourceFolder url="file://$MODULE_DIR$/generated_src" isTestSource="false" generated="true"/>
      <sourceFolder url="file://$MODULE_DIR$/generated_testSrc" isTestSource="true" generated="true"/>
</content>
```

**Comment by tjwilson90** [comment]
This is from `./gradlew idea` after cleaning a repo with `git clean -fdx`

**Comment by CRogers** [comment]
Fixed by #260
